src/pspm_get_txt.m

```diff
@@ -1,10 +1,11 @@
-function [sts, import, sourceinfo] = pspm_get_txt(datafile, import)
+function [sts, import, sourceinfo] = pspm_get_txt(datafile, import, delimiter)

```

Comment:
Again as I pointed it out last time, I prefer to be consistent in functions names and arguments, for different data file type there should be only `datafile` and `import` as argument (look at `pspm_get_eyelink`, `pspm_get_physlog`, etc.) So I propose to put this `delimiter` option into the `import` structure, or something like that.

Reply:
I remember being conflicted on putting it in import for some reason but I'll take another look, I was likely just a bit confused

Reply:
Ah yes, so the `import` seems to me to be associated with a specific channel, not the file. Is that right? It would mean something like adding the delimiter to all import cells, then just taking the first and considering it gospel. Shall I do that? It's not particularly nice but I think that's the best way without an extra arg

Reply:
Or just put it on the first cell in the import cell array, it's hacky either way but just on first I guess is a smidgen less memory

Reply:
Yes that's correct, I think we can do it like for the `eyelink_trackdist` in the `pspm_get_eyelink` function where we are including it in every channel, it's a bit redundant but it's kind of logical, so what you can do is to include the delimiter in every channel of the `import` structure